plugins/modules/nd_policy_cam_statistics_hit_counts.py

```diff
@@ -190,6 +190,7 @@ def main():
     filter_by_attributes = nd.params.get("filter_by_attributes")
     output_csv = nd.params.get("output_csv")
 
+    hit_count_pair = ""
```

Comment:
is this needed by some checks in CI?

Reply:
seems you responded to similar question in ansible-aci regarding sanity and unbound variables.